lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -40,7 +40,7 @@
             {'path': '/usr/local/bin/brew', 'name': 'homebrew'},
             {'path': '/sbin/apk', 'name': 'apk'},
             {'path': '/usr/sbin/pkg', 'name': 'pkgng'},
-            {'path': '/usr/sbin/swlist', 'name': 'HP-UX'},
```

Comment:
don't remove swlist, but we should discuss priority

Reply:
@bcoca I looked for this and did found a binary swdepot which was hardlinked to swlist and all the other sw* tools, but not sure what version this is. (using forums)

I don't mind keeping swlist though. No need to add multiple IMO

Reply:
Reverted.

Reply:
the module itself doesnt use a swdepot binary (online i've only found refs to the other 3) so keeping swlist and pointing to swdepot module seems fine with me.

Reply:
This was my reference: https://community.hpe.com/t5/System-Administration/Everything-about-usr-sbin-sw-is-identical-Can-this-be-conpressed/td-p/2447105#.XFB9X1VKjmg